### PR TITLE
Feature ckeditor5

### PR DESCRIPTION
**Pull request by osallou**
use ckeditor5 and allow to use multiple textareas with editor

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -85,6 +86,12 @@ Quote templates are configured the same way than invoice templates.
 * [space] fix pending accounts bug
 * [booking] on calendar entry display, allow no customer/no pricing (catch errors on computeDuration)
 * [core][booking] layout fixes on booking edition (periodic)
+<<<<<<< HEAD
+* [booking] compute duration with 2 digits per minute
+* [booking] weekarea calendar, use summary view by default
+* [booking] if multiple prices are in db, pick first
+=======
+>>>>>>> develop
```
Comment (gaetanHerve) [inline CHANGELOG.md:94]: merge conflict not properly resolved

File: CHANGELOG.md [hunk 2]
```diff
@@ -189,7 +196,12 @@ Quote templates are configured the same way than invoice templates.
 
 ## 2.3.1
 
+<<<<<<< HEAD
+* [core] fix upgrade for redis invoice numbers (1641387865_update_invoice_numbers_redis.php)
+  If already applied with 2.3.0, need to remove it from pfm_upgrades table and re-apply install
+=======
```
Comment (gaetanHerve) [inline CHANGELOG.md:202]: idem

**Comment by gaetanHerve** [comment]
got that in comnewsedit page: 
ckeditor5_error

**Comment by osallou** [comment]
I checked only comtile....

**Comment by osallou** [comment]
reason is there are multiple "content" div ids....

**Comment by gaetanHerve** [comment]
Could be nice to make text area resizeable if it's simple to do. Else, it's ok to me

**Comment by osallou** [comment]
textarea has no parameter for its size, would be better to specify expected number of lines....
(ckeditor or not)

**Comment by gaetanHerve** [comment]
ok. Anyway, textarea block automatically expands when a new line is needed

**Review by gaetanHerve — APPROVED** [review]

**Comment by osallou** [comment]
@gaetanHerve  when you fill the textarea, it expends in size

**Pull request merged**